### DISCUSSION TITLE
Documentation.

### DISCUSSION THREAD

**Discussion by N3UE** [winder/Universal-G-Code-Sender#2376] [General]
### Description

I do promote UGS on a facebook group and try to help people with problems, but there is a lack of documentation.
an example is there is no information on Autoleveler. Kind of hard to help someone without any docs!

**Comment by breiler** [comment]
The wiki is open for anyone to edit and contribute to:
https://github.com/winder/Universal-G-Code-Sender/wiki/Usage

**Comment by N3UE** [comment]
It would be nice to be able to edit and contribute... but one would really
have to know just how it works first!

**Comment by breiler** [comment]
I'll move this to the discussions instead of an issue. Otherwise this issue will never be closed as there is always need for more documentation.

You mentioned the autoleveler, what else do you see is needed?

**Comment by N3UE** [comment]
Off hand I can't remember. The Autoleveler is the current issue. That should be good for now...
